paper-server/patches/sources/net/minecraft/world/entity/projectile/arrow/AbstractArrow.java.patch

```diff
@@ -1,5 +1,13 @@
 --- a/net/minecraft/world/entity/projectile/arrow/AbstractArrow.java
 +++ b/net/minecraft/world/entity/projectile/arrow/AbstractArrow.java
+@@ -2,6 +_,7 @@
+ 
+ import com.google.common.collect.Lists;
+ import com.mojang.serialization.Codec;
++import io.papermc.paper.configuration.type.number.IntOr;
```

Comment:
Can you inline this with a fully qualified name?
Diff in the import section is quite bad for us during updates as those tend to get shifted around a *lot*.

The rest of the logic lgtm.

Feel free to open a docs PR for this as well if you want to, otherwise one of our docs team can pick that up.